src/portod.cpp

```diff
@@ -908,7 +908,7 @@ static int MasterMain(bool respawn) {
     std::shared_ptr<TEpollLoop> ELoop = std::make_shared<TEpollLoop>();
     TError error = ELoop->Create();
     if (error)
-        return error;
```

Comment:
Here was conversion from bool to int, but explicitly return of EXIT_FAILURE will be better IMHO.